modules/ROOT/pages/troubleshooting.adoc

```diff
@@ -209,7 +209,7 @@ However, if you happened to do this, it is possible to recover.
  $ sudo ostree commit --consume --link-checkout-speedup --orphan --selinux-policy=/ /ostree/repo/tmp/selinux-fix
  $ sudo restorecon -Rv /var
  $ sudo restorecon -Rv /etc
- $ sudo ostree admin deploy fedora-atomic:fedora/30/x86_64/{variant}
+ $ sudo ostree admin deploy fedora:fedora/35/x86_64/{variant}
```

Comment:
Should we change this to 36? Although I'm fine with having 35 there..

Reply:
I didn't do it as it will always kind of always need to be updated for each release. Maybe I can convert it to use the variables to simplify the update.